.travis.yml

```diff
@@ -14,7 +14,7 @@ services:
 
 branches:
   only:
-    - /^8.x/
+    - /^8.8.x/
```

Comment:
The `branches` travis command matches the github branch name, which is why Travis is not being triggered for this PR.